content/install/server/apache.es.md

```diff
@@ -0,0 +1,60 @@
++++
+date = "2017-04-15T14:39:04+02:00"
+title = "Configuración con Apache"
+url = "setup-with-apache"
+
+[menu.install]
+  identifier = "setup-with-apache"
+  parent = "install_server"
+  weight = 5
++++
+
+Ésta guía provee una vista corta para instalar un servidor de Drone detrás de un servidor Apache2. Ésta es una configuración de ejemplo:
+
+```nohighlight
+ProxyPreserveHost On
+
+RequestHeader set X-Forwarded-Proto "https"
+
+ProxyPass /ws/ ws://localhost:8000/ws/
+ProxyPassReverse /ws/ ws://localhost:8000/ws/
+
+ProxyPass / http://127.0.0.1:8000/
+ProxyPassReverse / http://127.0.0.1:8000/
+```
+
+Debes tener los siguientes módules de Apache instalados:
```

Comment:
It says "módules", I'm sure it's just a typo. It should either be "módulos" or by taking the english word "modules".